README.md

```diff
@@ -1,2 +1,79 @@
 # Wall-E
 Srishti 2019
+
+![IMG_242133](https://user-images.githubusercontent.com/48998778/55190041-1ee14e00-51c5-11e9-9a4d-d7ac8141bacb.jpg)
+
+
+
+
+## Abstract
+Our main idea was to create a bot which can pick garbage directly from dustbin.
+
+
+## Motivation
+In this present era, people live a very busy life. In such a situation, a person will always look for the way to save
+time.So introducing the robot, WALL-E that can collect waste directly from dustbins and  then dispose the waste to the area which is already predefined. 
+
+
+
+## Mechanical Aspect of the Design
+
+
+Our bot has differential chasis with  three wheels out of which one is castor wheel and the other two motors are connected with two motors which provide movement to the bot.
+With the help of these wheels, the bot can move in all directions.The hands were operated using three dc motors.The "up-down" motion of hands were provided  using worm and spur gear mechanism.The hands were able to converge or diverge with the help of pulley and belt mechanism.
+The body was made using channels and plywood whereas the head was 3D printed.
+
+
+## Electronics Aspect of the Design
+We used two microcontrollers namely Arduino Uno and Arduino Nano, two L298n motor drivers, cytron motor driver,dc motors, servo motor,  bluetooth module and two LED Matrix MAX7219 for the this bot.Uno was used for movements of wheels and hands whereas nano was used for operating the head including LED Matrix used as the eyes of our bot, WallE.
```

Comment:
Write the following section in points as a numbered list
1. Mechanical Aspect
2. Electronic aspect